test/conformance/adapters/hip/test_context.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2022-2023 Intel Corporation
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "context.hpp"
+#include "hip_fixtures.h"
+#include "queue.hpp"
+
+using urHipContextTest = uur::urDeviceTest;
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urHipContextTest);
+
+// https://sep5.readthedocs.io/en/latest/ROCm_API_References/
+// HIP_API/Context-Management.html#_CPPv419hipCtxGetApiVersion8hipCtx_tPi
+constexpr int HIP_DRIVER_API_VERSION = 4;
+
+TEST_P(urHipContextTest, ActiveContexts) {
+
```

Comment:
this newline is inconsistent between this and the other test

Reply:
👍